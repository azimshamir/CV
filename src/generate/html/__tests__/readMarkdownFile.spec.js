import readMarkdownFile from "../readMarkdownFile";
// import markdown from "./markdown.md";

const MARKDOWN_OPTIONS_DEFAULT = {
  encoding: "utf8",
};

const MARKDOWN_AS_HTML = `<h1 id="hello-world">Hello World</h1>
<p>This is some <em>markdown</em>.</p>
<p><strong>FOR REAL!</strong></p>
`

describe("readMarkdownFile", () => {
  it("generates correct html", () => {
    expect(
      readMarkdownFile(
        "src/generate/html/__tests__/markdown.md", 
        MARKDOWN_OPTIONS_DEFAULT
      )
    ).toBe(
      MARKDOWN_AS_HTML
    );
  });
});
