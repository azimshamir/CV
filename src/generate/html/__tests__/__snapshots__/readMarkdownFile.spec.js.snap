// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`readMarkdownFile generates correct html 1`] = `
"<h1 id=\\"hello-world\\">Hello World</h1>
<p>This is some <em>markdown</em>.</p>
<p><strong>FOR REAL!</strong></p>
"
`;
